docs/eventing-api.md

```diff
@@ -273,7 +273,22 @@ knative.dev/pkg/apis/duck/v1.KReference
 <td>
 <em>(Optional)</em>
 <p>DeadLetterChannel is a KReference and is set by the channel when it supports native error handling via a channel
-Failed messages are delivered here.</p>
+Failed messages are delivered here.
+Deprecated, see DeadLetterSinkUri insteam</p>
```

Comment:
insteam -> instead

Reply:
Yeah, this was me seeing a refactor and forgetting to stash things before doing a cleanup.